draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -781,6 +781,22 @@ Verifiers need to ensure that they are not using expired or revoked keys
 for signature verification using reasonable and appropriate means for the given
 key-distribution method.
 
+## Selective Disclosure without Holder Binding
+
+The Issuers and the Verifiers should be aware that when Holder Binding is not present,
+any entity that is not the original receiver can be presenting the Combined Format for Presentation.
+When doing so, that entity may be applying selective disclosure to the SD-JWT
+that has already been previously selectively disclosed.
+
+For example, a device manufacturer might produce an SD-JWT
+containing information about upstream and downstream supply chain contributors.
+Each supply chain party can verify only the claims that were selectively disclosed to them
+by an upstream party, and they can choose to apply selective disclosure to those claims
+when presenting to a downstream party.
+
+In some scenarios this behavior could be desirable,
+but if it is not, the Issuers are encouraged to apply Holder Binding.
```

Comment:
```suggestion
In some scenarios this behavior could be desirable,
but if it is not, Verifiers are encouraged to enforce Holder Binding.
```
maybe not use a MUST here?

Reply:
or "need to"?

Reply:
"need to" would work too. I was just wanting to avoid introducing the capitalized normative language in this section that is about general conceptual guidance.

Reply:
```suggestion
In some scenarios this behavior could be desirable,
but if it is not, Verifiers need to enforce Holder Binding.
```

Reply:
maybe "Issuers ought to (provide for / leverage / require ), and Verifiers ought to enforce Holder Binding".

Reply:
I think the point is **the Issuers** need to enforce holder binding if it does not want anyone to be able to forward sd-jwts all over. it's not about verifiers ignoring holder binding when it is present.

(I am ok with `need to enforce` but it should be issuers, not verifiers)

Reply:
How can an Issuer enforce holder binding when the Verifier can always opt not to check it?

Reply:
Issuer needs to be aware that without binding, the credential is basically a Netflix password in a big family.
Verifier (Netflix) needs to be aware that accepting a password with no binding maybe means the current user is not the original account holder.

Sorry for trollish analogy, but they both need to be given guidance (Issuer and Verifier).

Reply:
Yes, both need guidance, but the Issuer cannot _enforce_ holder binding. Only the Verifier can. The Issuer should _support_ holder binding though.

Reply:
The issuer "enables" holder binding", the verifier "enforces" it.